### PR TITLE
3f5a9a7kq

### PR DESCRIPTION
**Pull request by thlurte**
Following features have been integrated from different branches

- [x] Mentor's home page (UI/UX) `86b1u16yh` @NushaMBZ 
- [x] Reset Password Page (UI/UX) `86b1tvmtu` @s92068606
- [x] Login Page (UI/UX) `86b1tvmry` @s92068606 
- [x] Student's home Page (UI/UX)   `86b1tvpc8` @s92068606 
- [x] Registration Form and Email Design `86b1u1981` @NaRkIsJaHaN 
- [x] Retrieve and Display historical activity `86b1u14r5` `86b1u14r5` @NaRkIsJaHaN `Minhaj (minhajnazeer395@gmail.com)` 
- [x]  Mentor registering Student Form `86b1u14wh` @AynulAmna  
- [x]  Activity Logging Model `86b1tvpe1` @NaRkIsJaHaN  
- [x]  Add & Manage Activity `86b1tvpke` @LakshananS 
- [x]  JWT Authentication `86b1u0xkk` @Mohamedshazan 
- [x] Registration email with password reset link `86b1u14d8` @AM-ASKY-97  
 
Additionally the following features have been introduced.

- [x] Email format updated  
- [x] Mapping students with mentors  
- [x] Getting the students mapped to the relevant mentor in to the dropdown list 
- [x] Storing who is selected at the dropdown list 
- [x] Display the calendar of the relevant user selected in the dropdown list
- [x] Mentors reviewing approve/reject students activity   
- [x] Bulk report generation feature
- [x] Individual report generation feature

### CODE REVIEW

**Review by NaRkIsJaHaN — APPROVED** [review]

**Review by s92068606 — APPROVED** [review]

**Review by NushaMBZ — APPROVED** [review]

**Review by AynulAmna — APPROVED** [review]

File: server/prisma/seed.ts [hunk 1]
```diff
@@ -5,9 +5,10 @@ const prisma = new PrismaClient();
 
 async function main() {
   // Clear existing data in the reverse order of dependencies
+  await prisma.activity.deleteMany({});
+  await prisma.mentorActivity.deleteMany({});        
   await prisma.mentorFeedback.deleteMany({}); 
   await prisma.report.deleteMany({});          
-  await prisma.activity.deleteMany({});        
   await prisma.mentorship.deleteMany({});     
   await prisma.user.deleteMany({});
```
Comment (bsaranga) [inline server/prisma/seed.ts]: Remove the empty query object `{}` and call the `deleteMany` method without arguments.

File: server/prisma/seed.ts
Comment (bsaranga) [inline server/prisma/seed.ts:1]: Remove the current `require` based imports and add the following:

```
import { PrismaClient } from "@prisma/client";
import * as bcrypt from 'bcrypt-ts'
```

Rename the file extension to `.mjs` and update the seed script in `package.json`

**Review by bsaranga — CHANGES_REQUESTED** [review]
Attend to changes.

File: server/prisma/seed.ts (hunk 1 above)
Comment (thlurte) [reply server/prisma/seed.ts]: the empty objects have been removed.

File: server/prisma/seed.ts
Comment (thlurte) [reply server/prisma/seed.ts:1]: as requested, the changes have been made

**Review by bsaranga — APPROVED** [review]

**Pull request merged**